contracts/array.scrypt

```diff
@@ -0,0 +1,127 @@
+
+contract Array {
```

Comment:
👍 

I was planning to do this. We can wrap it into the standard library later for dynamic array.